rdt/__init__.py

```diff
@@ -42,10 +38,13 @@ def get_demo(num_rows=5):
         pd.DataFrame
     """
     # Hard code first five rows
-    login_dates = pd.Series([
-        '2021-06-26', '2021-02-10', 'NAT', '2020-09-26', '2020-12-22'
-    ], dtype='datetime64[ns]')
-    email_optin = pd.Series([False, False, False, True, np.nan], dtype='object')
+    login_dates = pd.Series(
+        ['2021-06-26', '2021-02-10', 'NAT', '2020-09-26', '2020-12-22'],
+        dtype='datetime64[ns]',
+    )
```

Comment:
anyway we can configure it to allow the paranthesis closing on an indented line like
```python
x = func(
    param1, param2, param3)
```

Reply:
Looking through [this page](https://docs.astral.sh/ruff/settings/#format_indent-style), I did not find any settings for this. ruff (which is replacing black) is meant to be:
> Black is the uncompromising Python code formatter. By using it, you agree to cede control over minutiae of hand-formatting. In return, Black gives you speed, determinism, and freedom from pycodestyle nagging about formatting. You will save time and mental energy for more important matters.